source/common/config/well_known_names.cc

```diff
@@ -95,7 +95,8 @@ TagNameValues::TagNameValues() {
   addRe2(SSL_CIPHER_SUITE, R"(^cluster\.<NAME>\.ssl\.ciphers(\.(<CIPHER>))$)", ".ssl.ciphers.");
 
   // cluster.[<route_target_cluster>.]grpc.(<grpc_service>.)*
-  addTokenized(GRPC_BRIDGE_SERVICE, "cluster.*.grpc.$.**");
+  // (?:\.\w*){2} matches the method name and metric name (if present)
```

Comment:
Thanks for reviewing. I updated since your comment so this is slightly out of date.

The non-capture groups (now simplified) are meant to "count backwards" the number of segments from the end. This was the only way I could think of to capture a gRPC service name with arbitrary number of dots in the service name.